components/TimeSeries.vue

```diff
@@ -84,7 +84,22 @@ const accordionStyle = {
 // We need each chart to have a higher z-index than the next one so that the exporting context menu is always on top and clickable.
 // Also, they should be at least 3 so that they are above .accordion-button:focus
 const zIndex = (Object.keys(appStore.timeSeriesData!).length - props.index) + 3;
-const yUnits = props.seriesId === "dead" ? "deaths" : "cases"; // TODO: Make this depend on a 'units' property in metadata. https://mrc-ide.myjetbrains.com/youtrack/issue/JIDEA-117/
+
+let yUnits: string; // TODO: Make this depend on a 'units' property in metadata. https://mrc-ide.myjetbrains.com/youtrack/issue/JIDEA-117/
+switch (props.seriesId) {
+  case "hospitalised":
+    yUnits = "in need of hospitalisation";
+    break;
+  case "dead":
+    yUnits = "deaths";
+    break;
+  case "vaccinated":
+    yUnits = "vaccinated";
+    break;
+  default:
+    yUnits = "cases";
+    break;
```

Comment:
It might be safer to just let the unit be empty, though we shouldn't ever really hit the breakpoint.